packages/components/tooltip/src/content.vue

```diff
@@ -93,10 +93,14 @@ const transitionClass = computed(() => {
 const persistentRef = computed(() => {
   // For testing, we would always want the content to be rendered
   // to the DOM, so we need to return true here.
-  if (process.env.NODE_ENV === 'test') {
-    if (!process.env.RUN_TEST_WITH_PERSISTENT) {
-      return true
-    }
+  if (typeof process === 'undefined') {
+    return true
+  }
+  if (
+    process.env.NODE_ENV === 'test' &&
+    !process?.env.RUN_TEST_WITH_PERSISTENT
+  ) {
+    return true
```

Comment:
I believe that if the `process` does not exist, the original logic should be followed instead of directly returning `true`.
```ts
if (
  typeof process !== 'undefined'
  && ...
  && ...
)
```